### PR TITLE
feat: Add beacons/POIs for territories with queued war timers

### PR DESCRIPTION
**Pull request by kristofbolyai**
image
image

### CODE REVIEW

File: common/src/main/java/com/wynntils/models/territories/TerritoryAttackTimer.java [hunk 1]
```diff
@@ -1,28 +1,38 @@
...
         this.minutes = minutes;
         this.seconds = seconds;
-        this.defense = "Unknown";
+        this.defense = null;
     }
 
     public String asString() {
-        return ChatFormatting.GRAY + territory + ChatFormatting.YELLOW + " (" + defense + ")" + ChatFormatting.AQUA
-                + " " + timerString();
+        ChatFormatting defenseColor = this.defense == null ? ChatFormatting.YELLOW : this.defense.getDefenceColor();
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/models/territories/TerritoryAttackTimer.java:31]: Noticing we're use `defense` in some places and `defence` in others, I think we should be using `defense`. Definitely for a seperate pr though as it's used a lot elsewhere

**Review by ShadowCat117 — APPROVED** [review]

File: common/src/main/java/com/wynntils/models/territories/TerritoryAttackTimer.java (hunk 1 above)
Comment (magicus) [reply common/src/main/java/com/wynntils/models/territories/TerritoryAttackTimer.java:31]: This is a recurring issue. :-( I think Wynncraft tried to standardize on one of them some time ago, and I think it was "defence". Imho we should use what they have in the name of attributes in their API.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/territories/TerritoryAttackTimer.java:31]: One could also say, use MC source names (mojmap).

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/territories/TerritoryAttackTimer.java:31]: > This is a recurring issue. :-( I think Wynncraft tried to standardize on one of them some time ago, and I think it was "defence". Imho we should use what they have in the name of attributes in their API.

Unfortunately I think Wynn uses a mix of both

Comment (magicus) [reply common/src/main/java/com/wynntils/models/territories/TerritoryAttackTimer.java:31]: > Unfortunately I think Wynn uses a mix of both

They have done so historically but I believe they recently standardized on one of them. Was what I was trying to say. :-)

**Pull request merged**